CartoDB/cartodb
# Show labels in the custom choropleth legend

Closes #11015 

### How to test this

1. Create a map.
2. Select aggregation by pixel.
3. Add a legend.
4. Change the left and right labels.

The labels should appear in the legend.

This fix depends on https://github.com/CartoDB/cartodb.js/pull/1534. Please, make sure you update your javascript dependencies before testing this fix.

![legend](https://cloud.githubusercontent.com/assets/4933/21144431/d917de1e-c14a-11e6-8847-2b0a26efe353.gif)
